packages/ui/src/components/charts/energy-consumption/energy-consumption-chart.tsx

```diff
@@ -42,7 +42,7 @@ export function EnergyConsumptionChart({ data, referencePoints, aggregation }: P
                     );
                 },
             }}
-            xAxes={{ dataKey: "timestamp", name: "Vergangene Zeit " + computeTimestampLabel(aggregation, false) }}
+            xAxes={{ dataKey: "timestamp" }}
```

Comment:
Das sollte am besten drin bleiben
```suggestion
                        xAxes={{ dataKey: "timestamp", name: "Vergangene Zeit " + computeTimestampLabel(aggregation, false) }}
```

Reply:
Kann ich auch gerne wieder reinnehmen aber fand es irgendiwe unübersichtlich